JavaAssignment/src/ankitaG/Program2.java

```diff
@@ -0,0 +1,22 @@
+package ankitaG;
+
+public class Program2 {
+	int studentrollno;
+	String studentName;
+	float studentMarks;
+	
+	void studentDetails(int rno, String name, float marks) {
+		studentrollno = rno;
```

Comment:
Again why are you initialize instance variable?  remove this because you are not using this method.